### PR TITLE
[arm cartesian motion] added rosparams for arm cart and joint velocity limits

### PR DESCRIPTION
**Pull request by abhishekpadalkar**

### CODE REVIEW

File: mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(mcr_arm_cartesian_control)
-
+add_compile_options(
+	  -O3
```
Comment (alex-mitrevski) [inline mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: Why is this necessary?

File: mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h [hunk 2]
```diff
@@ -26,6 +26,10 @@ class Arm_Cartesian_Control
     std::vector<double> upper_joint_limits;
     std::vector<double> lower_joint_limits;
 
+    double max_lin_frame_velocitiy = 0.1;  // m/s
```
Comment (alex-mitrevski) [inline mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h]: There's a typo in the variable name

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]

File: mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt (hunk 1 above)
Comment (abhishekpadalkar) [reply mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: It is not necessary, but optimization will enhance the performance.

File: mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h (hunk 2 above)
Comment (abhishekpadalkar) [reply mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h]: this was copy paste : ). I will change it.

File: mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt (hunk 1 above)
Comment (alex-mitrevski) [reply mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: Then please add a comment about that in the CMakeLists.

Comment (argenos) [reply mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: Agree with this, clarify what it's for and what the -03 means.

File: mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h (hunk 2 above)
Comment (abhishekpadalkar) [reply mcr_manipulation/mcr_arm_cartesian_control/common/include/mcr_arm_cartesian_control/arm_cartesian_control.h]: Fixed.

File: mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt (hunk 1 above)
Comment (abhishekpadalkar) [reply mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: Added a short description and possible effects of it. It is highly unlikely that code will crash as -O3 is very standard optimization. For more info https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html

Comment (alex-mitrevski) [reply mcr_manipulation/mcr_arm_cartesian_control/CMakeLists.txt]: The explanation was really about someone looking at the CMakeLists in one year and not wondering why the flag was required in this particular package. ;)

**Review by alex-mitrevski — APPROVED** [review]

**Review by sthoduka — APPROVED** [review]

**Pull request merged**